src/components/formio/select.tsx

```diff
@@ -35,6 +35,12 @@ export interface SelectProps<
   onChange?: (event: {target: {name: string; value: ValueType}}) => void;
 }
 
+export interface Option {
+  value: string;
+  label: React.ReactNode;
+  description?: string;
+}
```

Comment:
See the remark on line 20 :grimacing: 

The shape of react select options doesn't really matter, anything goes as long as you can point react-select to which property holds the actual 'value' and provide a callback/component to render the label given an option. By default, it looks at `value` and `label`, but it's actually super flexible.

Certain fields are just different than others and have different types of options that are relevant for them, and that's okay.

Reply:
I tried this initially with valueProperty="slug" which I thought was causing issues, but tried it again just now and it works, so I've removed the transformation. I do need this or some other type for the `select/preview.tsx` options though